src/templates/next/components/shared/Card.tsx

```diff
@@ -0,0 +1,119 @@
+import { SingleCardProps } from "~/common/InfoCards"
+import { BiRightArrowAlt } from "react-icons/bi"
+
+interface KeyStatisticsProps extends SingleCardProps {
+  variant?: "horizontal" | "vertical"
+  className?: string
+}
+
+const ImageComponent = ({
+  src,
+  alt,
+  className,
+}: {
+  src: SingleCardProps["imageUrl"]
+  alt: SingleCardProps["imageAlt"]
+  className?: string
+}) => {
+  return (
+    <div className={`aspect-h-1 aspect-w-1 overflow-hidden ${className}`}>
+      <img
+        src={src}
+        alt={alt}
+        className="h-full w-full object-cover object-center"
+      />
+    </div>
+  )
+}
+
+const TextComponent = ({
+  title,
+  text,
+  buttonLabel,
+  buttonUrl,
+  className,
+}: {
+  text: SingleCardProps["text"]
+  title: SingleCardProps["title"]
+  buttonLabel: SingleCardProps["buttonLabel"]
+  buttonUrl: SingleCardProps["buttonUrl"]
+  className?: string
+}) => {
+  return (
+    <div className={`p-7 flex flex-col flex-grow gap-3 h-fit ${className}`}>
+      {title && (
+        <h4 className="text-content text-left text-lg font-semibold sm:text-2xl">
+          {title}
+        </h4>
+      )}
+      <div className="text-content text-left text-base sm:text-lg grow">
+        {text}
+      </div>
+      {buttonLabel && buttonUrl && (
+        <a
+          className="flex flex-row gap-1 items-center font-semibold text-secondary text-md sm:text-lg"
+          href={buttonUrl}
+          target={buttonUrl?.startsWith("http") ? "_blank" : undefined}
+          rel={
+            buttonUrl?.startsWith("http")
+              ? "noopener noreferrer nofollow"
+              : undefined
+          }
+        >
+          {buttonLabel}
+          <BiRightArrowAlt className="size-6" />
+        </a>
```

Comment:
just wondering if `/next/Button` with `variant=link` and rightarrow icon would work here?
if doesn't fit style then nvm

Reply:
Yeah it does, replaced in  https://github.com/isomerpages/isomer-components/pull/38/commits/41f06d4ad7dcea032d2acfc3b74620a679420cb6!